src/util/getAllConfigs.js

```diff
@@ -1,21 +1,19 @@
 import defaultConfig from '../../stubs/defaultConfig.stub.js'
 import { flagEnabled } from '../featureFlags'
-import { flatMap, get } from 'lodash'
 
 export default function getAllConfigs(config) {
-  const configs = flatMap([...get(config, 'presets', [defaultConfig])].reverse(), (preset) => {
-    return getAllConfigs(preset)
-  })
+  const configs = (config && config.presets ? config.presets : [defaultConfig])
```

Comment:
Personally I find the reliance of truthiness and shortcircuiting stuff in these type of statements a bit obfuscating. We rely on `get` from lodash quite a lot throughout the codebase and I think making this sort of change everywhere is going to make the code quite a bit less declarative and harder to read.

I'd like some sort of abstraction around these sorts of "grab a key from an object but use this default if it's not there" operations which happen so much in the codebase. One very robust and very small library I've used for this in the past is `dlv`, perhaps we can just pull that in and start swapping these over?

https://github.com/developit/dlv

Reply:
by combining es2020 **optional Chaining** and **nullish coalescing** you get something as declerative as get or dlv:
```JS
config?.presets ?? [defaultConfig]
```

Since you already have **Babel** configured, only need to update **eslint** ecmaVersion.


Anyhow it's your decision.

Reply:
I'm good with that if you want to update the eslint config in this PR too then 👍🏻